aten/src/ATen/native/native_functions.yaml

```diff
@@ -4649,6 +4656,8 @@
   device_check: NoCheck
   device_guard: False
 
+- func: _efficientzerotensor(int[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
```

Comment:
I think you want to do the same as other factory functions:
- make it CompositeExplicitAutograd
- add it at the bottom of derivatives.yaml and make the output non-differentiable.

Reply:
Since it doesn't have a tensor input, I think I am not allowed to make it CompositeExplicitAutograd. Since now we also support forward mode AD, it might be useful to allow this to specify the output differentiability